ansible_collections/arista/avd/roles/eos_designs/python_modules/underlay/router_isis.py

```diff
@@ -90,3 +97,17 @@ def _is_type(self) -> str:
         if is_type not in ["level-1", "level-2", "level-1-2"]:
             is_type = "level-2"
         return is_type
+
+    @staticmethod
+    def ipv4_to_isis_system_id(ipv4_address):
```

Comment:
Even though we are not enforcing static type checker, I recommend you add them.
```suggestion
    def ipv4_to_isis_system_id(ipv4_address: str) -> str:
```